doc/_doxygen/groups.dox

```diff
@@ -22,6 +22,12 @@
 @{
 @}
 
+@brief Interfaces for USB hardware and associated standards.
+@defgroup usb_interface USB
```

Comment:
Maybe `usb_interfaces` ? to be consistent with io_interfaces, misc_interfaces, ... and "Interfaces for USB..." above.

Reply:
sure! addressed.